.github/workflows/code-health.yml

```diff
@@ -75,11 +75,13 @@ jobs:
         run: export resource_name=push_based_log_export && make generate-doc
       - name: Doc for search_deployment
         run: export resource_name=search_deployment && make generate-doc
+      - name: Doc for stream_processor
+        run: export resource_name=stream_processor && make generate-doc
       - name: Find mutations
         id: self_mutation
         run: |-
           git add .
-          git diff --staged --patch --exit-code > .repo.patch || echo "self_mutation_happened=true" >> "${GITHUB_OUTPUT}"
+          git diff -- . ':(exclude)*/docs/resources/stream_processor.md' --staged --patch --exit-code > .repo.patch || echo "self_mutation_happened=true" >> "${GITHUB_OUTPUT}"
```

Comment:
this change is needed so we could manually improve the description of the `state` attribute in the resource documentation